letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -53,22 +52,17 @@ class ApacheConfigurator(augeas_configurator.AugeasConfigurator):
     # pylint: disable=too-many-instance-attributes,too-many-public-methods
     """Apache configurator.
 
-    State of Configurator: This code has been tested under Ubuntu 12.04
-    Apache 2.2 and this code works for Ubuntu 14.04 Apache 2.4. Further
-    notes below.
-
-    This class was originally developed for Apache 2.2 and I have been slowly
-    transitioning the codebase to work with all of the 2.4 features.
-    I have implemented most of the changes... the missing ones are
-    mod_ssl.c vs ssl_mod, and I need to account for configuration variables.
-    This class can adequately configure most typical configurations but
-    is not ready to handle very complex configurations.
+    State of Configurator: This code has been been tested and built for Ubuntu
+    14.04 Apache 2.4 and it works for Ubuntu 12.04 Apache 2.2
 
-    .. todo:: Add support for config file variables Define rootDir /var/www/
-    .. todo:: Add proper support for module configuration
-
-    The API of this class will change in the coming weeks as the exact
-    needs of clients are clarified with the new and developing protocol.
+    .. todo:: Verify permissions on configuration root... it is easier than
+        checking permissions on each of the relative directories and less error
+        prone.
+    .. todo:: Write a server protocol finder. Listen <port> <protocol> or
+        Protocol <protocol>.  This can verify partial setups are correct
+    .. todo:: Add directives to sites-enabled... not sites-available.
+        sites-available doesn't allow immediate find_dir search even with save()
+        and load()
```

Comment:
You may want to reconsider if those TODO items are enough important for the API user to be put in the rendered docs. Especially the first two are a bit cryptic.